libmscore/property.cpp

```diff
@@ -675,6 +677,8 @@ QString propertyToString(Pid id, QVariant value, bool mscx)
                               return "whitekeys";
                         case GlissandoStyle::DIATONIC:
                               return "diatonic";
+                        case GlissandoStyle::CONTINUOUS:
+                              return "continuous";
                         case GlissandoStyle::CHROMATIC:
                               return "Chromatic";
```

Comment:
Hmm, why is this starting with an upper case? Not the fault of this PR but needs checking...

Reply:
Seems not to matter, as on read this is is assumed to be the default, but is strange and inconstent anyhow.

Reply:
Do we want to get this inconsistency fixed? Even if not really relevant and not for thos PR?

Reply:
Not particularly, I'm not confident that this wouldn't create a bunch of side effects

Reply:
See above, this should not matter at all, as that `Chromatic` is never used on rewading that property, it is just taken as the default. But indeed a bit out of the score of your PR.